venia-integration-tests/run-tests.js

```diff
@@ -78,20 +78,24 @@ const port = new URL(baseUrl).port;
 
 let dockerCommand = null;
 
+if (process.env.CI) {
```

Comment:
we still want to leave in the dockerImage above this line: `let dockerImage = "cypress/included:8.3.1";`

Reply:
let's also set a default here for the DockerRegistry. if `process.env.DockerRegistry` is not set then it will default to `undefined` which we don't want, so maybe instead of checking for `process.env.CI` we can check for `process.env.DockerRegistry` and then set the value of `dockerImage` to that value. I think `process.env.CI` is a bit too broad since many scripts will use that env var in CI environments but it won't necessarily indicate that a docker registry should be set.

Reply:
> let's also set a default here for the DockerRegistry

Just an empty string? Preserve existing behavior?
```js
dockerImage = (process.env.DockerRegistry || '') + dockerImage
```

Reply:
also a valid way of doing it. this is the same logic as is currently in place in this pr.